interpreter/exec/int.ml

```diff
@@ -221,6 +223,9 @@ struct
   let ge_s x y = x >= y
   let ge_u x y = cmp_u x (>=) y
 
+  let to_int_s = Rep.to_int
+  let to_int_u i = Rep.to_int i land (Rep.to_int Rep.max_int lsl 1) lor 1
```

Comment:
I'm a bit surprised this isn't implemented anywhere else... or is it that the other implementations depend on the int module?

Reply:
Yeah, so was I. I think we never needed this for plain int.